src/app/modules/investigation-report-di/components/investigation-report-di-add/investigation-report-di-add.component.ts

```diff
@@ -873,15 +879,13 @@ export class InvestigationReportDiComponent implements OnInit {
 
   //new add
   //method for onchanging compaintType dropdown
-  public onComplaintTypeSelect(args, complaintTypeId: string, comTypeName?: string) {
+  private onComplaintTypeSelect(args, complaintTypeId: string, comTypeName?: string) {

```

Comment:
is this method invoked from html?

Reply:
no..this method is only used in ts file..